admin_guide/sysctls.adoc

```diff
@@ -56,42 +56,42 @@ The following sysctls are known to be namespaced:
 
 Sysctls that are not namespaced are called _node-level_ and must be set
 manually by the cluster administrator, either by means of the underlying Linux
-distribution of the nodes (e.g., via *_/etc/sysctls.conf_*) or using a DaemonSet
-with privileged containers.
+distribution of the nodes, such as by modifying the *_/etc/sysctls.conf_* file,
+or by using a DaemonSet with privileged containers.
 
 [NOTE]
 ====
 Consider marking nodes with special sysctls as tainted. Only schedule pods onto
 them that need those sysctl settings. Use the
-link:http://kubernetes.io/docs/user-guide/kubectl/kubectl_taint/[Kubernetes _taints and toleration_ feature] to implement this.
+xref:../admin_guide/scheduling/taints_tolerations.adoc#admin-guide-taints[taints
+and toleration feature] to mark the nodes.
 ====
 
 [[safe-vs-unsafe-sysclts]]
-== Safe vs Unsafe Sysctls
+== Safe versus unsafe sysctls
 
 Sysctls are grouped into _safe_ and _unsafe_ sysctls. In addition to proper
 namespacing, a safe sysctl must be properly isolated between pods on the same
-node. This means that setting a safe sysctl for one pod:
+node. This means that if you set a sysctl as safe for one pod it must not:
 
-- must not have any influence on any other pod on the node,
-- must not allow to harm the node's health, and
-- must not allow to gain CPU or memory resources outside of the resource limits of
-a pod.
+- Influence any other pod on the node
+- Harm the node's health
+- Gain CPU or memory resources outside of the resource limits of a pod
 
 By far, most of the namespaced sysctls are not necessarily considered safe.
 
-For {product-title} 3.3.1, the following sysctls are supported (whitelisted) in
-the safe set:
+Currently, {product-title} supports, or whitelists, the following sysctls
+in the safe set:
 
 - *_kernel.shm_rmid_forced_*
 - *_net.ipv4.ip_local_port_range_*
```

Comment:
`net.ipv4.tcp_syncookies` is in the safe set as well

Reply:
@ingvagabund, how long has  `net.ipv4.tcp_syncookies` been in the safe set?

Reply:
@ingvagabund ^

Reply:
Since k8s 1.4: https://github.com/kubernetes/kubernetes/pull/27180

Reply:
https://github.com/kubernetes/kubernetes/pull/27180/files#diff-853d8d6fa2710e6a38f79cf40ba47f8bR44

Reply:
Here's a PR to make that change in older versions: https://github.com/openshift/openshift-docs/pull/11232